lib/Dialect/FIRRTL/Transforms/InferResets.cpp

```diff
@@ -1579,72 +1606,61 @@ void InferResetsPass::determineImpl() {
 ///   the existing port/wire/node as reset.
 /// - If the module already has a port with the reset's name:
 ///   - If the port has the same type as the reset domain, reuses that port.
```

Comment:
Possibly missing a change indicating that the name must match too?

Reply:
Thanks!